packages/ListBoxBrowser/src/components/Browser/Browser.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from "prop-types";
+
+const propTypes = {
+  isLoading: PropTypes.bool,
+};
+const defaultProps = {
+  isLoading: false,
+};
+
+export default function Browser() {
+  return null;
+}
+
+Browser.propTypes = propTypes;
+Browser.defaultProps = defaultProps;
+Browser.displayName = "ListBoxBrowser.Browser";
```

Comment:
What's the purpose of this component, is there a better naming for this? `Browser.Browser` sounds kind of weird

Reply:
is the right side of the component, honestly, I don't know if there is a better name for it. right now let you set a `isLoading` prop .

This component exists to split props on the subcomponent without overwhelms the main `<ListBoxBrowser />` with tons of props.

Reply:
`RightColumn` ? :D? But i'm not going to die on this hill, if can't think of something better than we can leave it

Reply:
yes @oscarkwan please don't we need you